Signal/src/ViewControllers/SignalsViewController.m

```diff
@@ -695,14 +787,18 @@ - (void)presentViewControllerWithBlock:(void (^)())presentationBlock animateDism
     }
 }
 
-#pragma mark - IBAction
+#pragma mark - Groupings
 
-- (IBAction)selectedInbox:(id)sender {
+- (void)selectedInbox
+{
+    self.segmentedControl.selectedSegmentIndex = 0;
```

Comment:
Previously the only way to switch between these "groups" was through the segmented control, so it wasn't necessary to sync it with these changes. Now we can switch from the "reminder" that I've just added.